library/core/class.controller.php

```diff
@@ -275,6 +277,10 @@ public function __construct() {
         $hsts = Gdn::factory('HstsModel');
         $this->_Headers[HttpStrictTransportSecurityModel::HSTS_HEADER] = $hsts->getHsts();
 
+        $cspModel = Gdn::factory(ContentSecurityPolicyModel::class);
```

Comment:
I'm trying to get rid of `Gdn::factory()` (it just uses the container internally at this point). https://github.com/vanilla/vanilla/issues/7459

The standard way for this is `Gdn::getContainer()->get()` instead.